src/bionics.cpp

```diff
@@ -2053,6 +2055,15 @@ void load_bionic( JsonObject &jsobj )
 
     new_bionic.fake_item = jsobj.get_string( "fake_item", "" );
 
+    new_bionic.weight_capacity_modifier = jsobj.get_float( "weight_capacity_modifier", 1.0 );
+    if( jsobj.has_int( "weight_capacity_bonus" ) ) {
```

Comment:
Please don't allow raw numbers here. Only load this via `read_from_json_string` as you do below. It will cause an error when the input is not a string and properly validate it as well.

A value like "1 kg" in JSON way easier to understand than a raw number 1000. Or what advantage do you see in loading from a raw integer?